ci/jenkins/jobs/macros.yaml

```diff
@@ -259,3 +259,12 @@
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           chmod +x ./ci/jenkins/test-rancher.sh
           ./ci/jenkins/test-rancher.sh --cluster-name rancher-test --testcase '{conformance_type}' --registry ${{DOCKER_REGISTRY}}
+
+- builder:
+     name: builder-stop-all
+     builders:
+       - shell: |-
+           #!/bin/bash
+           set -e
+           chmod +x ./ci/jenkins/stop-jobs.sh
+           ./ci/jenkins/stop-jobs.sh --pull-request "${ghprbPullId}" --jenkins "${JENKINS_URL}"
```

Comment:
Is JENKINS_URL a jenkins environment variable? I think we should use "${{JENKINS_URL}}".

Reply:
yes, but when I am using "${{JENKINS_URL}}" instead of this "${JENKINS_URL}" then its showing error:
`/tmp/jenkins4409063882437197647.sh: line 4: ${{JENKINS_URL}}: bad substitution`